kong/client_test.go

```diff
@@ -32,6 +32,19 @@ func TestKongStatus(T *testing.T) {
 	assert.NotNil(status)
 }
 
+func TestKongConfig(t *testing.T) {
+	RunWhenDBMode(t, "off")
+	assert := assert.New(t)
+
+	client, err := NewTestClient(nil, nil)
+	assert.NoError(err)
+	assert.NotNil(client)
+
+	config, err := client.Config(defaultCtx)
+	assert.NoError(err)
+	assert.NotNil(config)
```

Comment:
I think this would make sense to assert some structure of the returned config - e.g. if we go with returning `[]byte`, ensure it's a valid json in the test.

Reply:
Assertions added